tests/resources/config/structure_config.yml

```diff
@@ -1,15 +1,15 @@
 # The absolute path of the root of the target project
 # If none is provided your cwd will be used
-project_root:
+project_root: D://wrapper_writter/
```

Comment:
This hasnt broken travis but it probebly should be removed

Reply:
taken out